app/src/main/java/com/chess/clock/engine/TimeControlParser.java

```diff
@@ -198,42 +218,48 @@ public static int getLastTimeControlCheckIndex(Context context) {
      *
      * @return Default TimeControl list.
      */
-    public static ArrayList<TimeControl> buildDefaultTimeControlsList(Context context) {
+    public static ArrayList<TimeControlWrapper> buildDefaultTimeControlsList(Context context) {
         Log.i(TAG, "Building and saving default time control list");
 
-        ArrayList<TimeControl> timeControls = new ArrayList<TimeControl>();
+        ArrayList<TimeControlWrapper> timeControls = new ArrayList<TimeControlWrapper>();
 
         // Fischer blitz 5|0
         Stage fischerBlitzStage = new Stage(0, 300000);
         TimeIncrement fischerBlitzTI = new TimeIncrement(TimeIncrement.Type.FISCHER, 0);
         TimeControl fischerBlitzTC = new TimeControl("Fischer Blitz 5|0", new Stage[]{fischerBlitzStage}, fischerBlitzTI);
+        TimeControlWrapper fischerBlitzTCWrapper = new TimeControlWrapper(fischerBlitzTC, fischerBlitzTC);
 
         // Delay bullet 1|2
         Stage delayBulletStage = new Stage(0, 60000);
         TimeIncrement delayBulletTimeIncrement = new TimeIncrement(TimeIncrement.Type.DELAY, 2000);
         TimeControl delayBulletTimeControl = new TimeControl("Delay Bullet 1|2", new Stage[]{delayBulletStage}, delayBulletTimeIncrement);
+        TimeControlWrapper delayBulletTimeControlWrapper = new TimeControlWrapper(delayBulletTimeControl, delayBulletTimeControl);
 
         // Blitz 5|5
         Stage blitz55 = new Stage(0, 300000);
         TimeIncrement blitzTimeIncrement = new TimeIncrement(TimeIncrement.Type.FISCHER, 5000);
         TimeControl blitzTimeControl = new TimeControl("Fischer 5|5", new Stage[]{blitz55}, blitzTimeIncrement);
+        TimeControlWrapper blitzTimeControlWrapper = new TimeControlWrapper(blitzTimeControl, blitzTimeControl);
 
         // Fischer rapid 10|5
         Stage fischerRapidStage = new Stage(0, 600000);
         TimeIncrement fischerRapidTI = new TimeIncrement(TimeIncrement.Type.FISCHER, 5000);
         TimeControl fischerRapid = new TimeControl("Fischer rapid 10|5", new Stage[]{fischerRapidStage}, fischerRapidTI);
+        TimeControlWrapper fischerRapidWrapper = new TimeControlWrapper(fischerRapid, fischerRapid);
+
 
         // Tournament
         Stage tournamentStage1 = new Stage(0, 7200000, 40);
         Stage tournamentStage2 = new Stage(1, 3600000);
         TimeIncrement tournamentTI = new TimeIncrement(TimeIncrement.Type.DELAY, 5000);
         TimeControl tournamentTC = new TimeControl("Tournament 40/2hr, G60, 5s delay", new Stage[]{tournamentStage1, tournamentStage2}, tournamentTI);
+        TimeControlWrapper tournamentTCWrapper = new TimeControlWrapper(tournamentTC, tournamentTC);
 
-        timeControls.add(fischerBlitzTC);
-        timeControls.add(delayBulletTimeControl);
-        timeControls.add(blitzTimeControl);
-        timeControls.add(fischerRapid);
-        timeControls.add(tournamentTC);
+        timeControls.add(fischerBlitzTCWrapper);
```

Comment:
I'm not a fan of the fact that some of these use `TC` while some use `TimeControl` which I realize is not related to your work and this PR...just something I stumbled upon :)

Reply:
The naming scheme is fixed in 0faf160. All default time controls are named `TC`